h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_drf.R

```diff
@@ -19,7 +19,10 @@ test.model.generic.drf <- function() {
     print(generic_model)
     
     original_output <- capture.output(print(original_model))
+    
     generic_output <- capture.output(print(generic_model))
+    generic_output <- drop_model_parameters_from_printout(generic_output)
```

Comment:
Why are we dropping the parameters from printout ? I don't say the output of parameters must be 1:1 exactly the same as I did it with output, but I don't see where the presence of model parameters is tested in these R tests.

Currently, there is only one test for random forest. I'd prefer having tests for each supported algorithm - these tests are already created, we only need to adjust them. Or at worst copy&paste them, if the output capturing does not suit your needs. Is there something preventing us from doing it ?